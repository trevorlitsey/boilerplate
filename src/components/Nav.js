import React from 'react';
import FontAwesomeIcon from '@fortawesome/react-fontawesome';
import faGoogle from '@fortawesome/fontawesome-free-brands/faGoogle';
import styled from 'styled-components';

const Container = styled.div`
	width: 100%;
	display: flex;
	justify-content: space-between;
`

const Nav = () => (
	<nav className="navbar navbar-expand-lg navbar-dark bg-primary">
		<Container>
			<div className="left">
				<ul className="navbar-nav mr-auto">
					<a className="navbar-brand" href="#">~Boilerplate Rocks~</a>
					<li className="nav-item active">
						<a className="nav-link" href="#">Preview<span className="sr-only">(current)</span></a>
					</li>
					<li className="nav-item">
						<a className="nav-link" href="#">Snippets</a>
					</li>
				</ul>
			</div>
			<div className="right">
				<button type="button" className="btn btn-light"><FontAwesomeIcon style={{ marginRight: '6px' }} icon={faGoogle} />Sign in with Google</button>
			</div>
		</Container>
	</nav >
)



export default Nav;