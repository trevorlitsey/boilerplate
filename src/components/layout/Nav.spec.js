import React from 'react';
import { shallow } from 'enzyme';

import Nav from './Nav';

describe('Nav', () => {

	it('should render without crashing', () => {
		const wrapper = renderNav();
		expect(wrapper).toMatchSnapshot();
	})

})



// -----------
function renderNav(props = {}) {
	const propsToUser = {
		location: {
			hash: '',
			key: 'jt5a16',
			pathname: '/snippets',
			search: ''
		}
	}

	return shallow(<Nav {...propsToUser} />);
}