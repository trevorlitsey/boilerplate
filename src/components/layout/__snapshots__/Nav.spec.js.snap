// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav
    location={
      Object {
        "hash": "",
        "key": "jt5a16",
        "pathname": "/snippets",
        "search": "",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <styled.div>
        <div
          className="left"
        >
          <ul
            className="navbar-nav"
          >
            <Link
              className="navbar-brand"
              replace={false}
              to="/"
            >
              Boilerplate
            </Link>
            <li
              className="nav-item false"
            >
              <Link
                className="nav-link"
                replace={false}
                to="/"
              >
                Preview
              </Link>
            </li>
            <li
              className="nav-item active"
            >
              <Link
                className="nav-link"
                replace={false}
                to="/snippets"
              >
                Snippets
              </Link>
            </li>
          </ul>
        </div>
        <div
          className="right"
        >
          <button
            className="btn btn-light"
            onClick={[Function]}
            type="button"
          >
            <FontAwesomeIcon$1
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Object {
                  "icon": Array [
                    488,
                    512,
                    Array [],
                    "f1a0",
                    "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                  ],
                  "iconName": "google",
                  "prefix": "fab",
                }
              }
              listItem={false}
              mask={null}
              name=""
              pull={null}
              pulse={false}
              rotation={null}
              size={null}
              spin={false}
              style={
                Object {
                  "marginRight": "6px",
                }
              }
              symbol={false}
              transform={null}
            />
            Sign in with Google
          </button>
        </div>
      </styled.div>,
      "className": "navbar navbar-expand-sm navbar-dark bg-primary",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <div
            className="left"
          >
            <ul
              className="navbar-nav"
            >
              <Link
                className="navbar-brand"
                replace={false}
                to="/"
              >
                Boilerplate
              </Link>
              <li
                className="nav-item false"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/"
                >
                  Preview
                </Link>
              </li>
              <li
                className="nav-item active"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/snippets"
                >
                  Snippets
                </Link>
              </li>
            </ul>
          </div>,
          <div
            className="right"
          >
            <button
              className="btn btn-light"
              onClick={[Function]}
              type="button"
            >
              <FontAwesomeIcon$1
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Object {
                    "icon": Array [
                      488,
                      512,
                      Array [],
                      "f1a0",
                      "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                    ],
                    "iconName": "google",
                    "prefix": "fab",
                  }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size={null}
                spin={false}
                style={
                  Object {
                    "marginRight": "6px",
                  }
                }
                symbol={false}
                transform={null}
              />
              Sign in with Google
            </button>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul
              className="navbar-nav"
            >
              <Link
                className="navbar-brand"
                replace={false}
                to="/"
              >
                Boilerplate
              </Link>
              <li
                className="nav-item false"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/"
                >
                  Preview
                </Link>
              </li>
              <li
                className="nav-item active"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/snippets"
                >
                  Snippets
                </Link>
              </li>
            </ul>,
            "className": "left",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Link
                  className="navbar-brand"
                  replace={false}
                  to="/"
                >
                  Boilerplate
                </Link>,
                <li
                  className="nav-item false"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/"
                  >
                    Preview
                  </Link>
                </li>,
                <li
                  className="nav-item active"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/snippets"
                  >
                    Snippets
                  </Link>
                </li>,
              ],
              "className": "navbar-nav",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Boilerplate",
                  "className": "navbar-brand",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Boilerplate",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="nav-link"
                    replace={false}
                    to="/"
                  >
                    Preview
                  </Link>,
                  "className": "nav-item false",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Preview",
                    "className": "nav-link",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Preview",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="nav-link"
                    replace={false}
                    to="/snippets"
                  >
                    Snippets
                  </Link>,
                  "className": "nav-item active",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Snippets",
                    "className": "nav-link",
                    "replace": false,
                    "to": "/snippets",
                  },
                  "ref": null,
                  "rendered": "Snippets",
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <button
              className="btn btn-light"
              onClick={[Function]}
              type="button"
            >
              <FontAwesomeIcon$1
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Object {
                    "icon": Array [
                      488,
                      512,
                      Array [],
                      "f1a0",
                      "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                    ],
                    "iconName": "google",
                    "prefix": "fab",
                  }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size={null}
                spin={false}
                style={
                  Object {
                    "marginRight": "6px",
                  }
                }
                symbol={false}
                transform={null}
              />
              Sign in with Google
            </button>,
            "className": "right",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <FontAwesomeIcon$1
                  border={false}
                  className=""
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        488,
                        512,
                        Array [],
                        "f1a0",
                        "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                      ],
                      "iconName": "google",
                      "prefix": "fab",
                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size={null}
                  spin={false}
                  style={
                    Object {
                      "marginRight": "6px",
                    }
                  }
                  symbol={false}
                  transform={null}
                />,
                "Sign in with Google",
              ],
              "className": "btn btn-light",
              "onClick": [Function],
              "type": "button",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "border": false,
                  "className": "",
                  "fixedWidth": false,
                  "flip": null,
                  "icon": Object {
                    "icon": Array [
                      488,
                      512,
                      Array [],
                      "f1a0",
                      "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                    ],
                    "iconName": "google",
                    "prefix": "fab",
                  },
                  "listItem": false,
                  "mask": null,
                  "name": "",
                  "pull": null,
                  "pulse": false,
                  "rotation": null,
                  "size": null,
                  "spin": false,
                  "style": Object {
                    "marginRight": "6px",
                  },
                  "symbol": false,
                  "transform": null,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "Sign in with Google",
            ],
            "type": "button",
          },
          "type": "div",
        },
      ],
      "type": [Function],
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <styled.div>
          <div
            className="left"
          >
            <ul
              className="navbar-nav"
            >
              <Link
                className="navbar-brand"
                replace={false}
                to="/"
              >
                Boilerplate
              </Link>
              <li
                className="nav-item false"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/"
                >
                  Preview
                </Link>
              </li>
              <li
                className="nav-item active"
              >
                <Link
                  className="nav-link"
                  replace={false}
                  to="/snippets"
                >
                  Snippets
                </Link>
              </li>
            </ul>
          </div>
          <div
            className="right"
          >
            <button
              className="btn btn-light"
              onClick={[Function]}
              type="button"
            >
              <FontAwesomeIcon$1
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Object {
                    "icon": Array [
                      488,
                      512,
                      Array [],
                      "f1a0",
                      "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                    ],
                    "iconName": "google",
                    "prefix": "fab",
                  }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size={null}
                spin={false}
                style={
                  Object {
                    "marginRight": "6px",
                  }
                }
                symbol={false}
                transform={null}
              />
              Sign in with Google
            </button>
          </div>
        </styled.div>,
        "className": "navbar navbar-expand-sm navbar-dark bg-primary",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <div
              className="left"
            >
              <ul
                className="navbar-nav"
              >
                <Link
                  className="navbar-brand"
                  replace={false}
                  to="/"
                >
                  Boilerplate
                </Link>
                <li
                  className="nav-item false"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/"
                  >
                    Preview
                  </Link>
                </li>
                <li
                  className="nav-item active"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/snippets"
                  >
                    Snippets
                  </Link>
                </li>
              </ul>
            </div>,
            <div
              className="right"
            >
              <button
                className="btn btn-light"
                onClick={[Function]}
                type="button"
              >
                <FontAwesomeIcon$1
                  border={false}
                  className=""
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        488,
                        512,
                        Array [],
                        "f1a0",
                        "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                      ],
                      "iconName": "google",
                      "prefix": "fab",
                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size={null}
                  spin={false}
                  style={
                    Object {
                      "marginRight": "6px",
                    }
                  }
                  symbol={false}
                  transform={null}
                />
                Sign in with Google
              </button>
            </div>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="navbar-nav"
              >
                <Link
                  className="navbar-brand"
                  replace={false}
                  to="/"
                >
                  Boilerplate
                </Link>
                <li
                  className="nav-item false"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/"
                  >
                    Preview
                  </Link>
                </li>
                <li
                  className="nav-item active"
                >
                  <Link
                    className="nav-link"
                    replace={false}
                    to="/snippets"
                  >
                    Snippets
                  </Link>
                </li>
              </ul>,
              "className": "left",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Link
                    className="navbar-brand"
                    replace={false}
                    to="/"
                  >
                    Boilerplate
                  </Link>,
                  <li
                    className="nav-item false"
                  >
                    <Link
                      className="nav-link"
                      replace={false}
                      to="/"
                    >
                      Preview
                    </Link>
                  </li>,
                  <li
                    className="nav-item active"
                  >
                    <Link
                      className="nav-link"
                      replace={false}
                      to="/snippets"
                    >
                      Snippets
                    </Link>
                  </li>,
                ],
                "className": "navbar-nav",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Boilerplate",
                    "className": "navbar-brand",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Boilerplate",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="nav-link"
                      replace={false}
                      to="/"
                    >
                      Preview
                    </Link>,
                    "className": "nav-item false",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Preview",
                      "className": "nav-link",
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": "Preview",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="nav-link"
                      replace={false}
                      to="/snippets"
                    >
                      Snippets
                    </Link>,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Snippets",
                      "className": "nav-link",
                      "replace": false,
                      "to": "/snippets",
                    },
                    "ref": null,
                    "rendered": "Snippets",
                    "type": [Function],
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <button
                className="btn btn-light"
                onClick={[Function]}
                type="button"
              >
                <FontAwesomeIcon$1
                  border={false}
                  className=""
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        488,
                        512,
                        Array [],
                        "f1a0",
                        "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                      ],
                      "iconName": "google",
                      "prefix": "fab",
                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size={null}
                  spin={false}
                  style={
                    Object {
                      "marginRight": "6px",
                    }
                  }
                  symbol={false}
                  transform={null}
                />
                Sign in with Google
              </button>,
              "className": "right",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <FontAwesomeIcon$1
                    border={false}
                    className=""
                    fixedWidth={false}
                    flip={null}
                    icon={
                      Object {
                        "icon": Array [
                          488,
                          512,
                          Array [],
                          "f1a0",
                          "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                        ],
                        "iconName": "google",
                        "prefix": "fab",
                      }
                    }
                    listItem={false}
                    mask={null}
                    name=""
                    pull={null}
                    pulse={false}
                    rotation={null}
                    size={null}
                    spin={false}
                    style={
                      Object {
                        "marginRight": "6px",
                      }
                    }
                    symbol={false}
                    transform={null}
                  />,
                  "Sign in with Google",
                ],
                "className": "btn btn-light",
                "onClick": [Function],
                "type": "button",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "border": false,
                    "className": "",
                    "fixedWidth": false,
                    "flip": null,
                    "icon": Object {
                      "icon": Array [
                        488,
                        512,
                        Array [],
                        "f1a0",
                        "M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z",
                      ],
                      "iconName": "google",
                      "prefix": "fab",
                    },
                    "listItem": false,
                    "mask": null,
                    "name": "",
                    "pull": null,
                    "pulse": false,
                    "rotation": null,
                    "size": null,
                    "spin": false,
                    "style": Object {
                      "marginRight": "6px",
                    },
                    "symbol": false,
                    "transform": null,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "Sign in with Google",
              ],
              "type": "button",
            },
            "type": "div",
          },
        ],
        "type": [Function],
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
